### DISCUSSION TITLE
Minestom is a seriously good server impl

### DISCUSSION THREAD

**Discussion by cody-quinn** [Minestom/Minestom#1151] [General]
Minestom is amazing

DONTCAR: sunglasses :E

**Comment by emortaldev** [comment]
aGREE:sunglasses:D

**Comment by ryleu** [comment]
TRU:sunglasses:E

**Comment by GoldenStack** [comment]
Bro

**Reply by KrystilizeNevaDies** [reply]
I got pinged in my emails with "New github discussion: Minestom is a seriously good server impl, bro"
